modules/nf-core/smncopynumbercaller/main.nf

```diff
@@ -34,19 +34,19 @@ process SMNCOPYNUMBERCALLER {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        SMNCopyNumberCaller: $VERSION
+        SMNCopyNumberCaller: "1.1.2"
     END_VERSIONS
     """
 
     stub:
     """
     mkdir out
-    touch out/${prefix}.tsv
-    touch out/${prefix}.json
+    touch out/smnstub.tsv
+    touch out/smnstub.json
```

Comment:
The whole point of stub is to run the modules as if they were actually run. So the output files should have the same name as they would have if the module was run